docs/client.rst

```diff
@@ -40,6 +40,10 @@ When no other parameters are provided, the following keyword arguments are confi
 - **ssl** By default False. If enabled will make the connection using SSL/TLS protocol.
 - **ssl_verify** By default True. If enabled, will verify if the server certificate is trustable.
 - **ssl_extra_ca** By default `None`. If provided will used as an extr CA file used during the certificate verify. Use together `ssl_verify` when needed.
+- **autodiscovery** By default, `False`. If enabled the client will periodically call `config get cluster` and update node list according to its output. Note, this is not a standard feature of memcached, but is necessary to
```

Comment:
We would need to update the Readme for telling that autodiscovery for AWS is supported, also in the docs when how clients are created would be interesting to describe this functionallity.

Reply:
Updated README.rst, could you clarify the second part?

Reply:
there is this documentation that would be nice to update too https://github.com/emcache/emcache/blob/master/docs/client.rst

Reply:
This was editing that file, but I also added an example of calling it.